docs/README.md

```diff
@@ -16,7 +16,7 @@ Opstrace clusters expose a horizontally scalable [Prometheus](https://prometheus
 You can point your existing Prometheus or [FluentD](https://fluentd.org)/[Promtail](https://github.com/grafana/loki/blob/master/docs/sources/clients/promtail/_index.md) instances to it.
 We also plan to support a wide variety of other APIs, such as the Datadog agent.
 
-Creating an Opstrace Cluster requires our [command-line interface](./docs/references/cli.md), which talks directly to your cloud provider.
+Creating an Opstrace Cluster requires our [command-line interface](./references/cli.md), which talks directly to your cloud provider.
```

Comment:
@triclambert @PatrickHeneise this change is not yet live! Let's do that :)

Reply:
I've redeployed the prod branch manually.  Please confirm the fix is picked up.  See also #125.